cmd/ddev/cmd/list_test.go

```diff
@@ -64,3 +67,28 @@ func TestDevList(t *testing.T) {
 	}
 
 }
+
+// TestDdevListContinuous tests the --continuous flag for ddev list.
+func TestDdevListContinuous(t *testing.T) {
+	assert := asrt.New(t)
+
+	// Execute "ddev list --continuous"
+	cmd := oexec.Command(DdevBin, "list", "--continuous")
+	var cmdOutput bytes.Buffer
+	cmd.Stdout = &cmdOutput
```

Comment:
I thought you'd probably use [testcommon.CaptureStdOut()](https://github.com/drud/ddev/blob/e5c368d284e647d6444065f085522581b258032c/pkg/testcommon/testcommon.go#L199) here.

Reply:
I wasn't entirely sure what that was for, and looking at the code, I still don't quite understand it. I was trying to capture the stdout from the child process.

Reply:
Forgot this was in the cmd section, not sure if testcommon.CaptureStdOut() works for a child.